{"name": "Изменение данных для не авторизованного пользователя", "status": "broken", "statusDetails": {"message": "KeyError: 'accessToken'", "trace": "self = <allure_commons._allure.fixture object at 0x1042623f0>, args = (), kwargs = {}\n\n    def __call__(self, *args, **kwargs):\n        self.parameters = func_parameters(self._fixture_function, *args, **kwargs)\n    \n        with self:\n>           return self._fixture_function(*args, **kwargs)\n\n/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/allure_commons/_allure.py:221: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\n    @pytest.fixture(scope='function')\n    def register_new_user_return_response():\n        with allure.step('Получение данных о зарегистрированном пользователе'):\n            data = api.create_new_user()\n    \n            yield data[1]\n    \n        with allure.step('Получение токена созданного пользователя'):\n>           access_token = data[1].json()[\"accessToken\"]\nE           KeyError: 'accessToken'\n\nconftest.py:21: KeyError"}, "parameters": [{"name": "changed_data", "value": "'{\"email\": data[0], \"password\": fake.password(), \"name\": data[2]}'"}], "start": 1721243855790, "stop": 1721243856129, "uuid": "8ac5311e-c1fc-4284-b5db-e5ced04f514e", "historyId": "a4498face3ee8877891caa54ff492e3c", "testCaseId": "4dfaca9359ac1c438ac0b28494690521", "fullName": "tests.test_udate_user_data.TestUpdateUserData#test_update_info_non_authorized_user", "labels": [{"name": "suite", "value": "Изменение данных пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestUpdateUserData"}, {"name": "host", "value": "NB-ISC-14346"}, {"name": "thread", "value": "40747-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_udate_user_data"}]}